src/lib/y2partitioner/widgets/columns/used_by.rb

```diff
@@ -0,0 +1,41 @@
+# Copyright (c) [2020] SUSE LLC
+#
+# All Rights Reserved.
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of version 2 of the GNU General Public License as published
+# by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, contact SUSE LLC.
+#
+# To contact SUSE LLC about this file by physical or electronic mail, you may
+# find current contact information at www.suse.com.
+
+require "yast"
+require "y2partitioner/widgets/columns/base"
+
+module Y2Partitioner
+  module Widgets
+    module Columns
+      # Widget for displaying the `Used by` column, usually the physical name of a block device
+      class UsedBy < Base
```

Comment:
I would mention that it only supports Bcache. Otherwise, I could think that it should work, for instance, for RAID members.